articles/iot-edge/how-to-create-transparent-gateway-windows.md

```diff
@@ -33,35 +33,44 @@ The following steps walk you through the process of creating the certificates an
 ## Prerequisites
 1.	[Install the Azure IoT Edge runtime][lnk-install-windows-x64] on a Windows device you want to use as the transparent gateway.
 
-1. Get OpenSSL for Windows. There are many ways you can install OpenSSL. The instructions here use vcpkg to accomplish this.
-   1. Download and install vcpkg with the following commands run from an administrator PowerShell. Navigate to a directory where you want to install OpenSSL, lets call this `$VCPKGDIR`.
+1. Get OpenSSL for Windows.
 
-   ```PowerShell
-   git clone https://github.com/Microsoft/vcpkg
-   cd vcpkg
-   .\bootstrap-vcpkg.bat
-   .\vcpkg integrate install
-   .\vcpkg install openssl:x64-windows
-   ```
+>[!NOTE]
+>If you already have OpenSSL installed on your Windows device, you may skip this step but 
+> please ensure that  `openssl.exe` is available in your `%PATH%` environment variable.
+
+  There are many ways you can install OpenSSL:
+   * Download and install any [third-party OpenSSL binaries](https://wiki.openssl.org/index.php/Binaries), for example, from [this project on SourceForge](https://sourceforge.net/projects/openssl/).
+   
+   * Download the OpenSSL source code and build the binaries on your machine by yourself or do this via [vcpkg](https://github.com/Microsoft/vcpkg). 
+     * The instructions here use vcpkg to download source code, compile and install OpenSSL on your Windows machine all in very easy to use steps.
+   
+     1. Navigate to a directory where you want to install vcpkg. From here on we'll refer to this as $VCPKGDIR. Follow the instructions to download and install [vcpkg](https://github.com/Microsoft/vcpkg).
+   
+     1. Once vcpkg is installed, from a powershell prompt, run the following command to install the OpenSSL package for Windows x64. This typically takes about 5 mins to complete.
+     
+     ```PowerShell
+     .\vcpkg install openssl:x64-windows
+     ```
+     
+     3. Add `$VCPKGDIR\vcpkg\packages\openssl_x64-windows\tools\openssl` to your `PATH` environment variable so that the `openssl.exe` file is available for invocation.
 
-   1. Set environment variable `OPENSSL_ROOT_DIR` to `$VCPKGDIR\vcpkg\packages\openssl_x64-windows` and also add `$VCPKGDIR\vcpkg\packages\openssl_x64-windows\tools\openssl` to your `PATH` environment variable.
+1. Navigate to the directory in which you want to work. From here on we'll refer to this as $WRKDIR.  All files will be created in this directory.
 
+   cd $WRKDIR
+   
 1.	Obtain the scripts to generate the required non-production certificates with the following command. These scripts help you create the necessary certificates to set up a transparent gateway.
 
    ```PowerShell
    git clone https://github.com/Azure/azure-iot-sdk-c.git
    ```
```

Comment:
Lines 64-66 should be indented with 3 spaces. This looks like 2?

Reply:
Its 3 spaces.